chapters/inheritance.tex

```diff
@@ -1036,7 +1057,7 @@ \subsection{Annotations for Redeclaration and Modification}\label{annotation-cho
 A a(x = 3 "PID");
 \end{lstlisting}
 
-It can also be applied to \lstinline!Boolean! variables to define a check box.
+The \lstinline!choiches! annotation can also be applied to \lstinline!Boolean! variables to define a check box.
```

Comment:
Good opportunity to remove bad space:
```suggestion
The \lstinline!choiches! annotation can also be applied to \lstinline!Boolean! variables to define a checkbox.
```

Reply:
Both forms are valid, so I don't see a need to change that. 
It seems that currently "check box" is more popular in US English and "checkbox" more popular in British English according to google ngrams

Reply:
Similarly I don't see a need for spaces everywhere in the document. 
The C++ standard states the following "we intentionally use a variety of different styles to emphasize that the core language does not enforce any particular style."

Reply:
OK, it wasn't my intention to turn US English into British.  Resolving.